### PR TITLE
Feature/new projects page

### PR DESCRIPTION
**Pull request by amyd11**
Story: https://jiraent.cms.gov/browse/ISPGBSS-842

### Changes
- added a new projects page to handle projects created with the new project form
- added tests for the projects page

### Testing
- visit `/packages` to confirm it shows the number of projects created via the NEW project form (`/packages/create`) by your logged in user
- confirm the page looks the same as the previous /projects page (except for the percentage bar on the project cards)
- verify pagination works as expected
- verify total number of projects displays the correct number in the subtitle text
- run the automated tests by running `docker exec -it govready-q-dev bash` and then running the tests. (new tests can be run via `./manage.py test project.tests.test_views.ProjectListViewTest`)

_Note: we are holding on the percent bar until we can do the work necessary for the controls model (new, existing, or whatever that is)_

### CODE REVIEW

File: project/views.py [hunk 1]
```diff
@@ -1,40 +1,69 @@
...
-    })
+    return render(
+        request,
+        "project/project-home.html",
+        {
+            "project": project,
+            "acronym": project.acronym,
+            "project_title": project.title,
+            "impact_level": project.impact_level,
+            # "controls_status_count": project.system.controls_status_count,
```
Comment (TMCamp) [inline project/views.py]: If we don't need these, let's just delete them. Since they are in version control we can always find them if we them.

**Comment by jauschalley** [comment]
For visibility, Amy and I figured out that the progress bar is still showing up on the project card and she is working to remove it.

**Review by jauschalley — APPROVED** [review]
Looks great!

**Review by TMCamp — APPROVED** [review]
Approved with a comment; if we have commented out code, we should just delete it since it will live in version control.

**Pull request merged**